app/sidekiq/host_content_update_job.rb

```diff
@@ -0,0 +1,33 @@
+class HostContentUpdateJob < DependencyResolutionJob
```

Comment:
I know this isn't doing a ton at the mo, but it'd be good to get some tests for this class, just so we're documenting behaviour

Reply:
ahh yeah good point I forgot to come back to this!